aten/src/ATen/TensorIteratorInternal.h

```diff
@@ -68,5 +72,79 @@ inline void serial_for_each(
   }
 }
 
+inline void serial_for_each(
```

Comment:
Maybe there is no better way to do it, but I'm not a huge fan of having to copy paste this fairly large chunk of code twice